.editorconfig

```diff
@@ -0,0 +1,3 @@
+[*]
+end_of_line = lf
+insert_final_newline = true
```

Comment:
This file should specify `root`

> root: special property that should be specified at the top of the file outside of any sections. Set to "true" to stop .editorconfig files search on current file.

http://editorconfig.org/

any nested ones shouldn't.